pysteps/io/interface.py

```diff
@@ -42,51 +43,55 @@ def get_method(name, method_type):
-        +--------------+-------------------------------------------------------+
-        | mch_metranet | metranet files in the MeteoSwiss (MCH) archive        |
-        |              | containing precipitation composites.                  |
-        +--------------+-------------------------------------------------------+
-        | opera_hdf5   | ODIM HDF5 file format used by Eumetnet/OPERA.         |
-        +--------------+-------------------------------------------------------+
-        | knmi_hdf5    |  HDF5 file format used by KNMI.                       |
-        +--------------+-------------------------------------------------------+
+        +--------------+------------------------------------------------------+
+        |     Name     |              Description                             |
+        +==============+======================================================+
+        | bom_rf3      |  NefCDF files used in the Boreau of Meterorology     |
+        |              |  archive containing precipitation intensity          |
+        |              |  composites.                                         |
+        +--------------+------------------------------------------------------+
+        | fmi_geotiff  |  GeoTIFF files used in the Finnish Meteorological    |
+        |              |  Institute (FMI) archive, containing reflectivity    |
+        |              |  composites (dBZ).                                   |
+        +--------------+------------------------------------------------------+
+        | fmi_pgm      |  PGM files used in the Finnish Meteorological        |
+        |              |  Institute (FMI) archive, containing reflectivity    |
+        |              |  composites (dBZ).                                   |
+        +--------------+------------------------------------------------------+
+        | mch_gif      | GIF files in the MeteoSwiss (MCH) archive containing |
+        |              | precipitation composites.                            |
+        +--------------+------------------------------------------------------+
+        | mch_hdf5     | HDF5 file format used by MeteoSiss (MCH).            |
+        +--------------+------------------------------------------------------+
+        | mch_metranet | metranet files in the MeteoSwiss (MCH) archive       |
+        |              | containing precipitation composites.                 |
+        +--------------+------------------------------------------------------+
+        | opera_hdf5   | ODIM HDF5 file format used by Eumetnet/OPERA.        |
+        +--------------+------------------------------------------------------+
+        | knmi_hdf5    |  HDF5 file format used by KNMI.                      |
+        +--------------+------------------------------------------------------+
+        | crri_eu      |  NetCDF SAF CRRI files                               |
```

Comment:
Maybe we should try to keep this in alphabetical order. 
(can you also adjust the position of the knmi_hdf5 entry, please?)